exporter/dynatraceexporter/internal/serialization/histogram.go

```diff
@@ -15,23 +15,14 @@
 package serialization // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/dynatraceexporter/internal/serialization"
 
 import (
-	"errors"
+	"go.uber.org/zap"
 
 	dtMetric "github.com/dynatrace-oss/dynatrace-metric-utils-go/metric"
 	"github.com/dynatrace-oss/dynatrace-metric-utils-go/metric/dimensions"
 	"go.opentelemetry.io/collector/pdata/pmetric"
 )
 
-func serializeHistogram(name, prefix string, dims dimensions.NormalizedDimensionList, t pmetric.MetricAggregationTemporality, dp pmetric.HistogramDataPoint) (string, error) {
-	if t == pmetric.MetricAggregationTemporalityCumulative {
-		// convert to delta histogram
-		// skip first point because there is nothing to calculate a delta from
-		// what if bucket bounds change
-		// TTL for cumulative histograms
-		// reset detection? if cumulative and count decreases, the process probably reset
-		return "", errors.New("cumulative histograms not supported")
-	}
-
+func serializeHistogramPoint(name, prefix string, dims dimensions.NormalizedDimensionList, dp pmetric.HistogramDataPoint) (string, error) {
```

Comment:
Since we now have internal functions which shouldn't be called by `serialization.go` maybe we should consider moving `histogram.go` to `histogram/histogram.go` in order to make `serializeHistogramPoint` private? `serializeHistogram` would be renamed `SerializeHistogram` so it could be called by `serialization.go`. Not a blocker just a suggestion to make the API between modules a bit cleaner

Reply:
This is already quite the PR. I added the changes you suggested on a separate branch: https://github.com/dynatrace-oss-contrib/opentelemetry-collector-contrib/compare/dt-updates-review...dynatrace-oss-contrib:opentelemetry-collector-contrib:move_files?expand=1. If you think it's fine we add them to this PR - not sure if the collector reviewers are happy with this though...